src/webapp/components/issues/IssueTable.tsx

```diff
@@ -239,9 +243,45 @@ export const IssueTable: React.FC<IssueTableProps> = React.memo(props => {
         );
     }, [filters, showStepFilter]);
 
-    return <ObjectsTable loading={loading} {...config} filterComponents={filterComponents} />;
+    return (
+        <ObjectsTable
+            loading={loading}
+            {...config}
+            customClasses={customClasses}
+            filterComponents={filterComponents}
+            onChangeSearch={undefined}
+        />
+    );
 });
 
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        toolbar: {
+            display: "flex",
+            flexWrap: "wrap",
+            paddingLeft: 0,
+            paddingRight: 0,
+            marginBottom: "1rem",
```

Comment:
suggestion: try some logical properties on this file for better adaptation to different languages. In this case:

`paddingInline: 0;
marginBlockEnd: "1rem"`

Have a look at the [wiki](https://github.com/EyeSeeTea/know-how/wiki/UI-Guide#using-css-logical-properties) for more context.